integration-cli/docker_cli_run_test.go

```diff
@@ -4284,6 +4284,17 @@ func (s *DockerSuite) TestRunCredentialSpecWellFormed(c *check.C) {
 func (s *DockerSuite) TestRunServicingContainer(c *check.C) {
 	testRequires(c, DaemonIsWindows, SameHostDaemon)
 
+	// This functionality does not exist in post-RS3 builds. Note we get the version number
+	// this way as test binaries are not manifested.
+	if runtime.GOOS == "windows" {
+		v, err := kernel.GetKernelVersion()
+		c.Assert(err, checker.IsNil)
+		build, _ := strconv.Atoi(strings.Split(strings.SplitN(v.String(), " ", 3)[2][1:], ".")[0])
+		if build >= 16299 {
```

Comment:
Shouldn't this be `>` since RS3 _is_ 16299?

Reply:
Fixed